### PR TITLE
GF-27 - Seeds for events added

### PR DESCRIPTION
**Pull request by MIKALAKE**

### CODE REVIEW

File: db/migrate/20220708081331_change_data_type_events.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ChangeDataTypeEvents < ActiveRecord::Migration[7.0]
+  def change
+    change_column :events, :date, :string
```
Comment (iamalextodoran) [inline db/migrate/20220708081331_change_data_type_events.rb]: Why did you change this into a string?

**Review by iamdavidpetric — APPROVED** [review]

File: app/controllers/v1/events_controller.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@ def index
     end
 
     def show
+      @event = Event.where("date > ?", Time.now).order("date ASC").first
       render json: @event
```
Comment (iamalextodoran) [inline app/controllers/v1/events_controller.rb]: Delete the `@`, you are not updating the global variable, you want to define a new local variable for this method (it can have the same name, `event`). And delete the `show` method from the `before_action` hook.

Comment (MIKALAKE) [reply app/controllers/v1/events_controller.rb]: Okay, but something is not working

**Review by iamalextodoran — APPROVED** [review]
